rag/utils/infinity_conn.py

```diff
@@ -121,7 +121,7 @@ def __init__(self):
                 connPool = ConnectionPool(infinity_uri)
                 inf_conn = connPool.get_conn()
                 res = inf_conn.show_current_node()
-                if res.error_code == ErrorCode.OK and res.server_status == "started":
+                if res.error_code == ErrorCode.OK and res.server_status in ["started", "alive"]:
```

Comment:
`server_status` possible valus are one of `starting`, `started`. There's no `alive`.